setup.py

```diff
@@ -40,7 +40,7 @@ def read(fname):
     include_package_data=True,
     python_requires=">=3.6",
     install_requires=requirements,
-    use_scm_version=use_scm,
+    #use_scm_version=use_scm,
```

Comment:
i saw in some other places that you've reverted to manual versioning... happy to discuss how this git-based versioning and auto-deployment works if you want sometime.  it can be a little confusing